programs/server/Server.cpp

```diff
@@ -385,7 +390,16 @@ bool getListenTry(const Poco::Util::AbstractConfiguration & config)
 {
     bool listen_try = config.getBool("listen_try", false);
     if (!listen_try)
-        listen_try = DB::getMultipleValuesFromConfig(config, "", "listen_host").empty();
+    {
```

Comment:
Please could you add comment what we are doing here and why?

Reply:
Still have no idea - this will only affect starting interserver - lines starting at 2153 - the logic is that if config listen_try is false, make it true if no listen_host is configured (which means this node doesn't serve anything but interserver communication). And this change will preserve this logic with composable protocols.